app/components/framework/FrameworkImageButton/index.tsx

```diff
@@ -19,8 +19,8 @@ const query = {
     fields: ['title', 'preview_image', 'id'],
 };
 
-interface Props {
-    frameworkId: number | string;
+export interface Props {
+    frameworkId?: number | string;
```

Comment:
Why change this typing? It's always expected to pass a frameworkId for FrameworkImageButton

Reply:
Better to use `frameworkId: number | string | undefined`